examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -413,6 +413,8 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
             }));
 
             ws.send(pointsJson);
+            console.time("sample response time");
```

Comment:
This is the place we send sample Json in point map, right?

Reply:
it should be. @baiqiushi can you confirm?